pkgs/os-specific/linux/procps-ng/default.nix

```diff
@@ -46,7 +46,7 @@ stdenv.mkDerivation rec {
   meta = {
     homepage = https://gitlab.com/procps-ng/procps;
     description = "Utilities that give information about processes using the /proc filesystem";
-    priority = 10; # less than coreutils, which also provides "kill" and "uptime"
+    priority = 11; # less than coreutils, which also provides "kill" and "uptime"
```

Comment:
Coreutils has 10. Which wasn't less before but certainly isn't now.
Does that cause conflicts?